File: pinyougou-interface/src/main/java/cn/itcast/core/service/ContentService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.ad.Content;
import entity.PageResult;

import java.util.List;

public interface ContentService {

	public List<Content> findAll();
	
	public PageResult findPage(Content content, Integer pageNum, Integer pageSize);
	
	public void add(Content content);
	
	public void edit(Content content);
	
	public Content findOne(Long id);
	
	public void delAll(Long[] ids);

    List<Content> findByCategoryId(Long categoryId);
}

File: pinyougou-web-portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){



	this.findByCategoryId = function(categoryId){
		return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
	}

    //网站前台商品分类显示
    //查询商品分类信息
    this.findItemCatList = function () {
        return $http.get("index/findItemCatList.do");
    }
});